QUEST_LV_0100.tsv

```diff
@@ -3898,9 +3898,9 @@ QUEST_LV_0100_20150317_003897	Defeat the Large Kepa that appear where the Search
 QUEST_LV_0100_20150317_003898	Go back to the Search Scout
 QUEST_LV_0100_20150317_003899	Return to Search Scout and tell him that the Large Kepa was defeated.
 QUEST_LV_0100_20150317_003900	Talk to Knight Commander Uska
-QUEST_LV_0100_20150317_003901	Knight Commander Uska is looking for people who came to Klaipeda for the dream of Goddess. Meet Uska and tell him about the dream of Revelation.
-QUEST_LV_0100_20150317_003902	Knight Commander Uska is looking for people who came to Klaipeda for the dream of Goddess. Meet Uska and tell him about the dream of Revelation.
-QUEST_LV_0100_20150317_003903	When ready, talk to Uska to go the Crystal mines as told by Goddess in the dream.
+QUEST_LV_0100_20150317_003901	Knight Commander Uska is looking for people who came to Klaipeda for the dream of Goddess. Meet Sir Uska and tell him about the dream of Revelation.
+QUEST_LV_0100_20150317_003902	Knight Commander Uska is looking for people who came to Klaipeda for the dream of Goddess. Meet Sir Uska and tell him about the dream of Revelation.
+QUEST_LV_0100_20150317_003903	When ready, talk to Sir Uska to go the Crystal mines as told by Goddess in the dream.
```

Comment:
I check the Korean text it ought to be: 
When ready, talk to Sir Uska about going to the Crystal Mine as told by the Goddess in the Bishop's dream.